components/automate-chef-io/content/docs/iam-v2-api-reference.md

```diff
@@ -74,227 +159,556 @@ You can see the actions contained in the `editor` role in the [Roles section]({{
+### Listing Policies
 
-```bash
-curl -s -H "api-token: $TOKEN" -d @policy.json https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/policies?pretty
-```
+The output includes both *Chef-managed* and *Custom* policies.
+Policies migrated from IAM v1 are included as *Custom* policies.
+Policy names that begin with `[Legacy]` are default policies migrated from v1.
+We recommend deleting these legacy policies and setting up new IAM v2 policies in their place.
 
-### Updating a Policy
+*In the browser:* **Settings**  >> **Policies**
 
-In the IAM beta, policies must be updated from the command line.
+### Getting a Policy
 
-* You can modify the *membership* of both Chef-managed and custom policies; see [Policy Membership]({{< relref "iam-v2-api-reference.md#policy-membership" >}}) for more information.
-* You can modify the *definition* of custom policies but not Chef-managed policies.
-* You can change the policy name, members, or statements, but you cannot change the policy ID.
-* Modifying any part of a policy resets *all* of its properties.
-  Properties that aren't included in the command are reset to empty values.
-  Thus, you must supply all of a policy's properties, not just the ones you wish to update.
+In order to fetch a policy on the command-line, you must know its ID.
+This may be obtained by first fetching the list of all policies and getting the `id` field of the policy of interest.
+For example, the ID for the default administrator policy is `administrator-access`.
 
-In order to update a policy, you must know its ID.
-Use the command from [Listing Policies]({{< relref "iam-v2-api-reference.md#listing-policies" >}}) to locate your policy ID.
+*In the browser:* **Settings**  >> **Policies** >> [select a policy]
 
-The simplest way to pass a policy is by storing it in a file; we used a sample `policy.json` in the previous section.
-To update that policy you still need to pass in the complete policy definition--plus its members--just as you did when creating it; any omitted properties will be erased from the policy!
+### Creating a Policy
 
-As an example, to update the example policy to keep the first policy statement allowing users list-access, while removing `editor` access, edit your `policy.json` file:
+Create a policy by composing JSON with all the necessary policy properties (see the beginning of the section above).
 
-```json
-{
-  "name": "Test Policy",
-  "id": "test-policy-1",
-  "members": ["team:local:alpha","team:ldap:beta"],
-  "statements": [
-    {
-      "effect": "ALLOW",
-      "actions": [
-        "iam:users:list"
-      ]
-    }
- ]
-}
-```
+### Updating a Policy
 
-To update your policy, replace the last component of the path in the command with the ID of the target policy.
+- You can modify the *membership* of both Chef-managed and custom policies; see [Policy Membership]({{< relref "iam-v2-api-reference.md#policy-membership" >}}) for more information.
```

Comment:
sometimes we capitalize `custom` and sometimes we don't . . . 😑

Reply:
Can't say I've been completely consistent, but my intent was to use `Custom` when referring to the property value and `custom` when it is just being an adjective.